notebooks/ch14/bias_modeling.md

```diff
@@ -204,15 +228,25 @@ Cross-validation provides a more accurate method of estimating our model error u
 
 The bias-variance tradeoff allows us to more precisely describe the modeling phenomena that we have seen thus far.
 
-Underfitting is typically caused by too much bias; overfitting is typically caused by too much variance.
+Underfitting is typically caused by too much bias; overfitting is typically caused by too much model variance.
 
-Collecting more data reduces variance. One recent trend is to select a model with low bias and high intrinsic variance (e.g. a neural network) and collect many data points so that the model variance is low enough to make accurate predictions. While effective in practice, collecting enough data for these models tends to require large amounts of time and money.
+Collecting more data reduces variance. In the extreme case that we can collect all the data for something, we would be able to create a model with zero variance- meaning that it predicts every single point correctly. One recent trend is to select a model with low bias and high intrinsic variance (e.g. a neural network) and collect many data points so that the model variance is low enough to make accurate predictions. While effective in practice, collecting enough data for these models tends to require large amounts of time and money.
 
 Collecting more data reduces bias if the model can fit the population process exactly. If the model is inherently incapable of modeling the population (as in the example above), even infinite data cannot get rid of model bias.
 
 Adding a useful feature to the data, such as a quadratic feature when the underlying process is quadratic, reduces bias. Adding a useless feature rarely increases bias.
 
-Adding a feature, whether useful or not, typically increases model variance. In order to increase a model's prediction accuracy, a new feature should decrease bias more than it increases variance.
+Adding a feature, whether useful or not, typically increases model variance. This is due to the fact that more features potentially will overfit the model:causing the model to be too specific for the training data, and not allow the model to generalize to potential unseen data. In order to increase a model's prediction accuracy, a new feature should decrease bias more than it increases variance. 
```

Comment:
This is due to the fact that more features potentially will overfit the model:causing the model to be too specific for the training data, and not allow the model to generalize to potential unseen data.

to 

...increases model variance because each new feature adds a new model parameter, increasing the number of possible models (...).